aten/src/ATen/BatchingRegistrations.cpp

```diff
@@ -527,6 +543,8 @@ TORCH_LIBRARY_IMPL(aten, Batched, m) {
 
   // view operations
   m.impl("chunk", chunk_batching_rule);
+  m.impl("tensor_split.sections", tensor_split_sections_batching_rule);
+  m.impl("tensor_split.indices", tensor_split_indices_batching_rule);
```

Comment:
@kurtamohler could you add some tests to test_vmap.py similar to https://github.com/pytorch/pytorch/blob/67889db8aac73335ba01a2b0a01099e33e8c31a1/test/test_vmap.py#L1240-L1259?

`_vmap_view_test` eventually calls into https://github.com/pytorch/pytorch/blob/67889db8aac73335ba01a2b0a01099e33e8c31a1/test/test_vmap.py#L650-L655 , the comments there might be helpful. Please shout if you have questions!

Reply:
Done! I used pretty similar test cases as `test_split`, but with a few differences.